src/js/vre/chapter31/config/form.js

```diff
@@ -60,14 +126,20 @@ const formConfig = {
       title: 'Military History',
       pages: {
         militaryHistory: {
-          path: 'military-information',
-          title: 'Military Information',
+          path: 'military-history',
+          title: 'Military History',
+          uiSchema: {
+            serviceHistory: serviceHistoryUI,
```

Comment:
Unless you are using these more than once, I think it's easier to read if they're in line here (not defined up above in a variable). That way you read them the context of the page config they're used on.

Reply:
Good point, relocated.